sdk/storage/CONTRIBUTION.md

```diff
@@ -0,0 +1,110 @@
+If you intend to contribute to the project, please make sure you've followed the instructions provided in the [Azure Projects Contribution Guidelines](https://opensource.microsoft.com/collaborate).
+
+## Project Setup on Windows
+On Windows, the Azure Storage development team uses Visual Studio so instructions will be tailored to that preference. However, any preferred IDE or other toolset should be usable.
+
+### Tool Dependencies
+* Visual Studio 2019 with C++ toolsets.
+* [Vcpkg](https://github.com/microsoft/vcpkg).
+* Clone the source code from GitHub.
+
+### Test Environment Dependencies
+
+#### Standard RA-GRS Storage Account to run Blob/File service tests
+You can leverage [this doc](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal) to create Standard RA-GRS Storage Account that can be used to run most of the test cases for Blob/File Share services.
+**Please ensure that you selected 'Standard' for performance, as this is to create the standard storage account.**
+**Please ensure that you selected `Read-access geo-redundant storage (RA-GRS)` to test the fall-back to secondary feature.**
+**Please ensure that in 'Data protection' tab, you need to enable the corresponding features for Storage's tests to pass. Recommendation is to check all boxes.**
+**Please ensure that in 'Advanced' tab, Customer-managed keys support are selected for CMK related tests.**
+
+After the Storage account is created, you can go to  Storage Account -> Settings -> Access keys, copy either the connection string in key1 or key2 and set it to system environment variable `STANDARD_STORAGE_CONNECTION_STRING`.
+
+#### DataLake Gen2 enabled Storage Account to run DataLake service tests.
+You can leverage [this doc](https://docs.microsoft.com/en-us/azure/storage/blobs/create-data-lake-storage-account) to create storage account to run test cases for DataLake Gen2 service.
```

Comment:
What specific section in the linked doc is relevant to this task? Please update all the links that point to large multi-section docs, to point to the exact relevant section, or at least name the section title in the url-text.